addons/project/views/project_task_views.xml

```diff
@@ -383,7 +383,7 @@
                                    widget="subtasks_one2many">
                                 <tree editable="bottom" decoration-muted="state in ['1_done','1_canceled']" open_form_view="True">
                                     <field name="allow_milestones" column_invisible="True"/>
-                                    <field name="display_in_project" column_invisible="True" force_save="1"/>
+                                    <field name="display_in_project" column_invisible="True"/>
```

Comment:
Actually your change here is not correct, removing the ``force_save`` here will just further break the feature. What the problem is is that the display_in_project field does not appear in the base task form and thus the ``force_save`` is not inside the activeFields in the record class. 
What you should do is add this line: 
```xml
<field name="display_in_project" invisible="True" force_save="1"/>
```
in line 339 under the ``project_id`` field.

Reply:
@pkyriakou  I have implemented as you said